tools/backend_manager.py

```
@@ -170,29 +171,40 @@ async def get_frontend_config():
-                        }
-                        for fe in self.detector.detected_frontends
-                    ]
-                },
-                "database": {
-                    "connected": True,  # TODO: Add actual DB health check
-                    "name": self.config.get('database', {}).get('name', 'ghost')
+            # Real DB connectivity probe (SELECT 1) via the shared DatabaseManager,
+            # rather than reporting a hardcoded "connected": True.
+            try:
+                db_connected = await get_db_manager().health_check_async()
+            except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)